configs/icds-14lakh-dimagivalues-jan2020.yml

```
@@ -0,0 +1,750 @@
+
+  preak_requests_per_15_min:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'peak_forms_per_15_min'
+      - 'peak_synclog_requests_per_15min'
+      - 'peak_other_requests_per_15min'
+
+
+  #Number of cases per user per month
+  cases_total:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 2000  # reduced from 8000 WCD figure
+    monthly_growth: 160  # guess (4 cases for every new person case)
+
+  # Number of cases updated per user per month
+  case_transactions:
+    model: 'derived_factor'
+    dependant_field: 'forms_monthly'
+    factor: 4
+  # Total Case Transaction
+  case_transactions_total:
+    model: 'cumulative'
+    dependant_field: 'case_transactions'
+    start_with: 1240906740
+
+  # Case indexes ever created
+  case_indices:
+    model: 'derived_factor'
+    dependant_field: 'cases_total'
+    factor: 1  # case index count (from SQL) / case count
+
+  # used for SMS & UCR calculations
+  person_cases_total:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 1000
+    monthly_growth: 28  # https://confluence.dimagi.com/display/ICDS/Case+Structure+and+Workflow#CaseStructureandWorkflow-CASCaseSizingEstimates
+
+  beneficiary_phone_numbers:
+    model: 'derived_factor'
+    dependant_field: 'person_cases_total'
+    factor: '{beneficiary_phone_number_seeding}'
+  phone_numbers:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'beneficiary_phone_numbers'
+      - 'users'  # 100% seeding
+      - 'ls_users'  # 100% seeding
+
+  pregancy_cases:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 6
+    monthly_growth: 1
+  child_cases:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 100
+    monthly_growth: 1
+  ledgers_preg:
+    model: 'derived_factor'
+    dependant_field: 'pregancy_cases'
+    factor: 5
+  ledgers_child:
+    model: 'derived_factor'
+    dependant_field: 'child_cases'
+    factor: 10  # was 35 but that seems too high
+  ledgers_total:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'ledgers_preg'
+      - 'ledgers_child'
+  ledger_transactions:
+    model: 'derived_factor'
+    dependant_field: 'ledgers_total'
+    factor: 2  # 2 transactions per ledger (ever)
+  ledger_updates_preg:
+    model: 'derived_factor'
+    dependant_field: 'pregancy_cases_baseline'  # active pregnancy cases
+    factor: 0.6
+  ledger_updates_child:
+    model: 'derived_factor'
+    dependant_field: 'child_cases_baseline'  # active child cases
+    factor: 0.6
+  ledger_updates_monthly:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'ledger_updates_preg'
+      - 'ledger_updates_child'
+
+  images_per_user:
+    model: 'date_range_value'
+    ranges:
+      - ['20180901', '20190301', 35]  # daily feeding
+      - ['20190401', '20190801', 60]
+      - ['20190901', '20200901', 650]  # daily feeding + THR + AG
```

"Images captured per user per month"

```suggestion
      - ['20190901', '20200901', 60]  # daily feeding + THR + AG
```